src/internal/dockertestenv/postgres.go

```diff
@@ -20,17 +20,17 @@ import (
 )
 
 const (
-	postgresPort  = 30228
-	pgBouncerPort = 30229
+	PostgresPort  = 30228
+	PGBouncerPort = 30229
 	maxOpenConns  = 10
 )
 
-func postgresHost() string {
+func PostgresHost() string {
 	return getDockerHost()
 }
 
-func pgBouncerHost() string {
-	return postgresHost()
+func PGBouncerHost() string {
+	return PostgresHost()
```

Comment:
I'm assuming this is for a test that needs to build a URL? It looks like we already have `dbutil.GetDSN()`, does that allow you to get around the refactoring you did here or is there more you need?